app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -281,6 +314,57 @@ fun EditBillAmounts() {
 
     }
 }
+@Composable
+fun SaveRemindButton() {
+    val context = LocalContext.current
+    val focusManager = LocalFocusManager.current
+    val shape = RoundedCornerShape(size = 16.dp)
+    var count by remember { mutableStateOf(0) }
+
+    IconButton(
+        onClick = {
+            if (count > 0) {
+                Toast.makeText(context, "Edit Saved!", Toast.LENGTH_SHORT).show()
+            } else {
+                Toast.makeText(context, "Saved!", Toast.LENGTH_SHORT).show()
+            }
+            count++
+            focusManager.clearFocus()
+        },
+
+
+        ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.Center
+        ) {
+            if (count > 0) {
+                Icon(
+                    painterResource(id = R.drawable.outline_check_circle_outline_24),
+                    contentDescription = "Saved Icon",
+                    tint = Color.Green,
+                    modifier = Modifier
+                        .size(50.dp)
+                )
+            } else {
+                Icon(
+                    painterResource(id = R.drawable.outline_data_saver_on_24),
+                    contentDescription = "Add Icon",
+                    tint = Color.Black,
+                    modifier = Modifier
+                        .size(50.dp)
+                )
+            }
+        }
+    }
+}
+
+@ExperimentalMaterialApi
+@Composable
+@Preview
+private fun SaveRemindButtonPreview() {
+    SaveRemindButton()
+}
```

Comment:
I incorporated the Remind Button Functionality within the Main Activity as I found the separate class file to be redundant. There is not enough code to prove the need to for a separate file and it serves important functionality INSIDE MainActivity.kt. https://unity-technologies.github.io/kotlin-guide/#declaring-multiple-classes-in-one-file